proc_guns/gun.py

```diff
@@ -1,77 +0,0 @@
-import affixes
-import random
-class Gun:
-    
-    def __init__(self, guntype, manufacturer, quality):  
```

Comment:
you're doing a whole lot all in one `__init__` method here. Looks like there are probably some logical behavior groups that could be split out into other functions!

Reply:
was trying not to split stuff into functions that were gonna get called once

Reply:
functions are good even if you're only using them once. you can test smaller units of behavior that way, and function names are a form of documentation that can be very helpful down the line in discerning what a chunk of code is/was supposed to be used for.

Reply:
chalk that up to DUELING BEST PRACTICES POORLY REMEMBERED